### PR TITLE
Setup Travis to run ui tests

### PR DESCRIPTION
**Pull request by ahmedre**
Experimenting based on @hosainnet's PR

### CODE REVIEW

**Comment by ozbek** [comment]
It seems to have stuck at "Waiting for emulator to start".

**Comment by ahmedre** [comment]
@ozbek it passes, it just takes a while.

@hosainnet is the `clean` needed for the connected test, or can i remove it? also, do you want me to merge this or do you want to clean up yours and i'll merge yours instead?

btw 2.0 - the reason this works is because of https://github.com/travis-ci/docs-travis-ci-com/issues/779 and https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943

basically you need to put tools twice.

**Comment by hosainnet** [comment]
@ahmedre I didn't have to put `tools` twice in the other PR when it worked initially, but if it's needed fair enough.

`clean` is there just for safety, similar to the unit test command

Feel free to merge this and close #764 :)

**Comment by ahmedre** [comment]
talked to Hosain offline - abandoning in favor of #764
